tests/src/test/scala/common/rest/WskRest.scala

```diff
@@ -1149,10 +1146,15 @@ class RunWskRestCmd() extends FlatSpec with RunWskCmd with Matchers with ScalaFu
   }
   val connectionContext = new HttpsConnectionContext(SSL.nonValidatingContext, Some(sslConfig))
 
-  def validateStatusCode(expectedExitCode: Int, statusCode: Int) = {
+  def validateStatusCode(expectedExitCode: Int, statusCode: Int, backupExpectedExitCode: Int = -1) = {
```

Comment:
What is the need for `backupExpectedExitCode`?

Reply:
There is a special case for invokeaction.
If blocking or result is set to true, 200 is an acceptable status code, no matter what expectedExitCode is provided. I will change the code of invokeaction to narrate this case.